lib/rules/web/admin_console/4.8/search.xyaml

```diff
@@ -17,18 +17,18 @@ click_resource_type_dropdown:
 send_type_filter:
   elements:
   - selector:
-      xpath: //input[@data-test-id='dropdown-text-filter']
+      xpath: //input[@placeholder='Select Resource']
     op: send_keys <resource_kind>
 search_resource_kind_only:
   element:
     selector:
-      xpath: //li//span[text()='<resource_kind>']
```

Comment:
It looks like we can just use `//span......`

Reply:
I have tried to just use //span,  event related case will fail due to too many elements are found on the background.